shared-fonts/style.css

```diff
@@ -0,0 +1,1500 @@
+@font-face {
```

Comment:
is that also the same icons font? looks like it is copied 3 times.
make sure you have only one and re-use it.